classes/events/Event.py

```diff
@@ -1,11 +1,8 @@
-from classes import World
```

Comment:
cloud solve the circular imports problem by importing the whole "classes" package

Reply:
Fixed in a PR in pipeline. don't bother to do the same monkey work twice